src/seedu/addressbook/commands/Reminder.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.commands;
+/**
+ * Add a reminder for user
```

Comment:
This comment doesn't follow the guidelines; please use the correct format for header comments.

Reply:
Sorry for the messy things, I'll update it as soon as possible. But my tutorial is on tomorrow, can I have more time to finish it?

Reply:
These messy things come out because I accidentally made all changes locally, really sorry about that fault. I will resubmit my work for 5.11. Is it acceptable I resubmit it by recess week?